libscpi/Makefile

```diff
@@ -1,9 +1,9 @@
 VERSION = 2.1.0
 LIBNAME = scpi
 
-CFLAGS += -Wextra -Wmissing-prototypes -Wimplicit -Iinc
+CFLAGS += $(MODE) -Wextra -Wmissing-prototypes -Wimplicit -Iinc
```

Comment:
What is `$(MODE)`? I can't find any documentation.

`CFLAGS` is constructed with `+=` so you can easily define your own flags on the command line.

If you don't like the build process, please provide separate PR just for it.

Reply:
see https://github.com/j123b567/scpi-parser/issues/141